src/it/java/org/checkstyle/base/AbstractItModuleTestSupport.java

```diff
@@ -247,10 +247,10 @@ protected final void verify(Checker checker,
             LineNumberReader lnr = new LineNumberReader(
                 new InputStreamReader(inputStream, StandardCharsets.UTF_8))) {
             int previousLineNumber = 0;
-            for (int i = 0; i < expected.length; i++) {
-                final String expectedResult = messageFileName + ":" + expected[i];
+            for (int index = 0; index < expected.length; index++) {
+                final String expectedResult = messageFileName + ":" + expected[index];
                 final String actual = lnr.readLine();
-                assertWithMessage("error message %s", i)
+                assertWithMessage("error message %s", index)
```

Comment:
that seems weird. `index` is index, so error message will be `error message 0` ?

I guess there is no need to pass index here and format string.

By the way, loop can be refactored to for-each loop

Reply:
> that seems weird. index is index, so error message will be error message 0 ?

I agree, many error messages in this class are strange/ hard to understand. Since this is an "example" can we keep this commit simple, and create an issue to improve error messages in test support?

Reply:
make sense to keep as is just for students

Reply:
lets agree to fix it in separate PR,  we have single case like this
```
14:36 $ ag -Q "assertWithMessage(\"error message %s\", i)"
src/it/java/org/checkstyle/base/AbstractItModuleTestSupport.java

```

Reply:
I did separate commit, but I think that it actually makes sense to keep regular for loop and similar (but improved) message. Example of new message:

```
error message 57 does not match expected message
expected: …galTokenText.java:161:29: Consider using …
but was : …galTokenText.java:160:29: Consider using …

```